### ISSUE TITLE
Run as real user jobs should set outputs_to_working_directory

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#6391]
Otherwise tools that write to outputs directly would fail without clear indication of why and what to do.

**Comment by bernt-matthias** [comment]
True. At least its already documented here: https://docs.galaxyproject.org/en/master/admin/cluster.html#configuration

Question is if there is a use case for which the value of the two variables differ. Otherwise one of the two variables could be removed... otherwise documentation (in galaxy.yml?) could be also improved.

Of course also the job_working_directory needs to be configured such that it is accessible by the real users. Also here documentation could be improved.
